Cargo.toml

```diff
@@ -16,19 +16,27 @@
 # under the License.
 
 [package]
-authors = ["Apache Software Foundation <dev@opendal.apache.org>"]
-categories = ["filesystem"]
-description = "OpenDAL: Access data freely, painlessly, and efficiently."
+name = "opendal"
+version = "0.30.3"
 edition = "2021"
-exclude = [".github/", "bindings/", "profiles/", "oli/", "tests/"]
+rust-version = "1.60" # MSRV for this project - please update while bump versions
+
+exclude = [
```

Comment:
We're now having the website source code also. I'd suggest we use include, or move the opendal core crate one level down (inner) as a follow-up.

Reply:
Nice idea, I will start a new PR for that.

Reply:
Tracked at https://github.com/apache/incubator-opendal/issues/1667